Lib/booleanOperations/booleanGlyph.py

```diff
@@ -78,7 +82,7 @@ def _get_clockwise(self):
 
     def _get_bounds(self):
         if self._bounds is None:
-            from robofab.pens.boundsPen import BoundsPen
+            from fontTools.pens.boundsPen import BoundsPen
```

Comment:
Yes,
https://github.com/behdad/fonttools/blob/master/Lib/fontTools/pens/boundsPen.py
and
https://github.com/robofab-developers/robofab/blob/master/Lib/robofab/pens/boundsPen.py
are the same afaict.